docker-compose-services/postgres/README.md

```diff
@@ -52,6 +52,58 @@ hooks:
 There are also another non-plain-text formats that `pg_dump` can generate, and you might need to work with them. If that's the case, there is also
 a `ddev pg_restore` command that will restore `.ddev/pgsql-db/postgresql.db.dump` into `db`.
 
+## Multiple databases
+
+* By default, DDEV adds a single database called `db`. But it's possible to create additional databases when you first initialize this service.
+
+* The offical way to add multiple databases is outlined on the [docker image](https://hub.docker.com/_/postgres) under the "Initialization scripts"
+
+To implement this in DDEV:
+
+* Add an additional volume to `./.ddev/docker-composer.postgres.yaml`
+
+  ```yaml
+      volumes:
+      ...
+      - "./pgsql-db:/docker-entrypoint-initdb.d"
+  ```
+
+* Create `./.ddev/pgsql-db` folder if it doesn't already exist
+
+  ```bash
+  mkdir ./.ddev/pgsql-db -p
+  ```
+
+* Add your *.sql,*.sql.gz, or *.sh files to `./.ddev/pgsql-db`. EG. Create a file called `./.ddev/pgsql-db/create_testing_database.sql` and add the following:
+
+  ```sql
+  CREATE DATABASE testing
+  ```
+
+* In fact, you could put any valid SQL connect such as creating or seeding the database.
+
+* Note: These files "are only run if you start the container with a data directory that is empty". IE. The first time you start the project. For exisiting projects, you must remove the volume first. Remember to backup the database first (Eg. `ddev pgsql_export`).
+
+### Removing a postgres volume
+
+  ```bash
+  # Display all docker volumes.
+  docker volume ls
+
+  # Stop DDEV.
+  ddev stop
+
+  # Remove 'example' project postgres volume.
+  docker volume rm ddev-example_postgres
+  ```
+
+* If you recieve a `Error response from daemon: ... volume is in use`, you need to stop DDEV first.
+
+### `ddev pgsql_export` auto-import
+
+* The file created by `ddev pgsql_export` is a great example of some of the commands you can use.
+* If this file exists, and the database data directory is empty (ie. you deleted the volume), the file will automatically by imported and populate your database!
```

Comment:
This is confusing, I think, pgsql_export will generate a db.sql that will be used for the primary "db", not sure how this relates

Reply:
Deleted